### PR TITLE
Calculator: Testing

### PR DESCRIPTION
**Pull request by henatan99**
In this Milestone the changes made are:
- Add snapshot react testing to DOM components
- Add tests to all components using react testing libraries 
- Add unit tests for calculate and operate logic modules
- Fix elsint errors 
- Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+
+## Quote Page 
+![Screenshot](src/Quote.png)
 
 ### Components 
 The project applies React's best practices and puts use of functional and class components. 
-It has Button, ButtonPanel and Display components which renders UI elements. Finally the App component renders ButtonPanel and Display components. 
-The App has state attributes, 
+It has Button, ButtonPanel and Display components which renders UI elements. Finally the CalculatorPage component renders ButtonPanel and Display components. 
+The CalculatorPage component has hooks.
```
Comment (bellom) [inline README.md:18]: README file not detailed enough for the newbie(non-technical individual), You should add steps to git clone your repo, add necessary installation to setup project locally and you need to update your live demo link.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @henatan99  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a one-issue left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md:18]: Thanks, @bellom. Now I made the requested changes.

**Comment by mimipeshy** [comment]
## STATUS: APPROVED 🥳 👯 
Hello team @henatan99  👋🏾 👋🏾 congratulations on making it this far in this milestone 🥇 

#### Highlights

- Your deployed app works correctly and the link is included in the README file ✔️ 
- You have worked on all changes requested by the previous TSE ✔️ 

With that, I approve of this project 😄 

Regards
__Peris Ndanu __
[GitHub ](https://github.com/mimipeshy)| [LinkedIn ](https://www.linkedin.com/in/peris-ndanu/)| [Twitter](https://twitter.com/pygirl254)

**Pull request merged**